src/aequitas/flow/methods/preprocessing/unawareness.py

```diff
@@ -0,0 +1,168 @@
+from typing import Optional
+
+import pandas as pd
+import numpy as np
+from scipy.stats import chi2_contingency
+
+from ...utils import create_logger
+from .preprocessing import PreProcessing
+
+
+class Unawareness(PreProcessing):
+    def __init__(
+        self, top_k: Optional[int] = 1, correlation_threshold: Optional[float] = None
+    ):
+        """Removes features that are highly correlated with the sensitive attribute.
```

Comment:
Add a note that we assume the sensitive attribute is a categorical variable, pls! 

something like Note: For this method, the vector `s` (protected attribute) is assumed to be categorical.